lib/chloride.rb

```diff
@@ -29,4 +30,31 @@ def self.go_execute(hostname, command)
       puts "We failed to run '#{command}' with error code #{remote_command.status} on #{hostname}"
     end
   end
+
+  def self.go_action(hostname, action, opts)
```

Comment:
This is a method I added so that I could test `net::scp` via the `bin/go_action` command.